node/coinstacks/solana/api/src/utils.ts

```diff
@@ -0,0 +1,59 @@
+import { EnrichedTransaction } from 'helius-sdk'
+import { Tx } from './models'
+import axios from 'axios'
+import { handleError } from '@shapeshiftoss/common-api'
+import { RPC_API_KEY, INDEXER_URL } from './constants'
+import axiosRetry, { isNetworkOrIdempotentRequestError } from 'axios-retry'
+
+export const axiosNoRetry = axios.create({ timeout: 5000, params: { 'api-key': RPC_API_KEY } })
+const axiosWithRetry = axios.create({ timeout: 10000, params: { 'api-key': RPC_API_KEY } })
+
+axiosRetry(axiosWithRetry, {
+  shouldResetTimeout: true,
+  retries: 5,
+  retryDelay: (retryCount, err) => {
+    console.log({ retryCount, err })
```

Comment:
```suggestion
```

Reply:
Fixed in [33a4238](https://github.com/shapeshift/unchained/pull/1053/commits/33a4238603fb52d1d0c78ab7639bde23265bee5f)